gravitee-apim-e2e/package.json

```
@@ -14,6 +14,7 @@
         "test:ui:e2e": "cypress open --config-file ui-test/integration/apim/e2e/cypress-config.json",
         "test:ui:platform": "cypress open --config-file ui-test/integration/platform/cypress-config.json",
         "test:api:mongo": "sh ./scripts/run-docker-compose.sh api-test mongo",
+        "test:bridge:mongo": "sh ./scripts/run-docker-compose.sh api-bridge mongo",
```

To be consistent with other `test:xxx:` scripts I would rename to:

```suggestion
        "test:api:bridge-mongo": "sh ./scripts/run-docker-compose.sh api-test-bridge mongo",
```